wicket-extensions-tester/src/test/java/org/apache/wicket/extensions/tester/AjaxLazyLoadPanelTesterTest.java

```diff
@@ -1,63 +1,65 @@
-/*

```

Comment:
`CRLF` endline has been changed to `LF`. Otherwise Git was refusing to move the files.

Reply:
Use https://github.com/apache/wicket/pull/708/files?diff=unified&w=1 to review without the whitespace changes